.github/workflows/triage-issues.yml

```diff
@@ -70,7 +70,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Lock Outdated Threads
-        uses: dessant/lock-threads@486f7380c15596f92b724e4260e4981c68d6bde6
+        uses: dessant/lock-threads@1621939cecf8586399a6b60d2a7af9469232b5b6
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
According to the docs, including this line is no longer needed. Should we get rid of it or keep it around to be safe?

```suggestion
```

Reply:
Going to keep it because we have it for the other two in this workflow, too